CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.31.0 Release
+- Release Date: May 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
```

Comment:
Should we add a note at the top of this list about recommending upgrading for servers not on v5.19, with a link to the blog post?

We should be on the lookout for opportunities to make this plug in ALL our documentation

Reply:
I can add a link to the blog afterwards when it has been published.